src/DayPicker.js

```diff
@@ -421,17 +421,15 @@ export default class DayPicker extends Component {
     return React.cloneElement(navbarElement, props);
   }
   renderDayInMonth(day, month) {
-    let dayModifiers = [];
-    if (DateUtils.isSameDay(day, new Date())) {
-      dayModifiers.push(this.props.classNames.today);
+    const { today } = this.props.classNames;
+    const propModifiers = Helpers.getModifiersFromProps(this.props);
+    const dayModifiers = Helpers.getModifiersForDay(day, propModifiers);
+    if (DateUtils.isSameDay(day, new Date()) && !propModifiers.hasOwnProperty(today)) {
```

Comment:
Please don't use `hasOwnProperty` this way or the lint will complain:
```diff
- propModifiers.hasOwnProperty(today)
+ Object.prototype.hasOwnProperty.call(propModifiers, today)
```
thanks!